### PR TITLE
enhanced OAM error message

### PR DESCRIPTION
**Pull request by asubedy**
Signed-off-by: asubedy <frexpe@pm.me>

**Description**

This PR fixes #

**Notes for Reviewers**

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

**Review by leecalcote — APPROVED** [review]
Good to go

File: nginx/error.go [hunk 1]
```diff
@@ -45,12 +45,14 @@ var (
...
+
+ 	// ErrParseOAMConfig represents the error which is
+ 	// generated during the OAM configuration parsing
+ 	ErrParseOAMConfig = errors.New(ErrParseOAMConfigCode, errors.Alert, []string{"error parsing the configuration"}, []string{"Error occured while parsing configuration in the request made by Meshery Server"}, []string{"Could not unmarshall OAM config recieved via ProcessOAM gRPC call into a valid Config struct"}, []string{"Check if Meshery Server is creating valid config for ProcessOAM gRPC call. This error should never happen and can be reported as a bug in Meshery Server. Also, confirm that Meshery Server and Adapters are referring to same config struct provided in MeshKit"})
```
Comment (jbyers19) [inline nginx/error.go:55]: Lines 51 and 55 should be split into multiple lines. They are too long and make the code less readable.

Comment (asubedy) [reply nginx/error.go:55]: like after the comments? @jbyers19

Comment (jbyers19) [reply nginx/error.go:55]: Yes. They are around 600 characters long. Max length depends on what the team prefers, but I recommend that they should be no longer than 120, ideally no longer than 80.

**Pull request merged**